docs/src/content/en/reference/workflows/execute.mdx

```diff
@@ -1,45 +1,19 @@
 ---
-title: "Reference: Workflow.execute() | Building Workflows | Mastra Docs"
-description: Documentation for the `.execute()` method in workflows, which executes a step with input data and returns the output.
+title: "Reference: Workflow.execute() | Workflows | Mastra Docs"
+description: Documentation for the `Workflow.execute()` method in workflows, which executes a workflow directly and returns the output.
 ---
 
 # Workflow.execute()
 
-The `.execute()` method executes a step with input data and returns the output, allowing you to process data within a workflow.
+The `.execute()` method executes a workflow directly and returns the output, allowing you to run a workflow without creating a separate run instance.
 
-## Usage
+## Usage example
 
-```typescript
```

Comment:
Same comment as for `docs/src/content/en/reference/workflows/create-run.mdx` file